src/Features/CSharp/Portable/Structure/Providers/EnumDeclarationStructureProvider.cs

```diff
@@ -24,9 +24,13 @@ internal class EnumDeclarationStructureProvider : AbstractSyntaxNodeStructurePro
             if (!enumDeclaration.OpenBraceToken.IsMissing &&
                 !enumDeclaration.CloseBraceToken.IsMissing)
             {
+                SyntaxNodeOrToken current = enumDeclaration;
+                var nextSibling = current.GetNextSibling();
+
                 spans.AddIfNotNull(CSharpStructureHelpers.CreateBlockSpan(
                     enumDeclaration,
                     enumDeclaration.Identifier,
+                    compressEmptyLines: !nextSibling.IsNode || nextSibling.AsNode() is BaseTypeDeclarationSyntax,
```

Comment:
hrmm. i would not expect type declarations to compress to each other FWIW.

Reply:
Collapse to Definitions does not collapse type nodes. If you manually collapse all, it will show them grouped.

Reply:
ah.  potentially worth a comment.  up to you.

Reply:
added a comment